node/src/chain_spec/turing.rs

```diff
@@ -72,7 +72,7 @@ pub fn turing_development_config() -> ChainSpec {
 				get_account_id_from_seed::<sr25519::Public>("Alice"),
 				endowed_accounts,
 				REGISTERED_PARA_ID.into(),
-				vec![b"AutomationTime".to_vec(), b"Balances".to_vec(), b"Democracy".to_vec()],
```

Comment:
removed these for dev cause it sucks to have to make a call each time to get them to just open up